src/modules/Workspaces/WorkspacesSnapshotTool/PwaHelper.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+namespace SnapshotUtils
+{
+    class PwaHelper
+    {
+    public:
+        std::map<std::wstring, std::wstring> pwaAumidToAppId;
+        std::vector<std::wstring> chromeAppIds;
+        std::map<std::wstring, std::wstring> pwaAppIdsToAppNames;
```

Comment:
Class members should always be private to minimize the chance of unintended access.